addons/account/tests/test_account_move_in_invoice.py

```diff
@@ -759,6 +759,47 @@ def test_in_invoice_line_onchange_taxes_1(self):
             'amount_total': 1384.0,
         })
 
+    def test_in_invoice_change_date_rounding_multi_currency(self):
+        new_currency_id = self.env['res.currency'].create({
+                'name': 'Test',
+                'symbol': 'T',
+                'rounding': 0.01,
+            })
+
+        self.env['res.currency.rate'].create([
+            {
+                'currency_id': new_currency_id.id,
+                'name': '2017-01-01',
+                'rate': 0.233661237937,
+            }, {
+                'currency_id': new_currency_id.id,
+                'name': '2016-01-01',
+                'rate': 0.233666697822,
+            }
+        ])
+
+        tax_20 = self.env['account.tax'].create({
+            'name': "tax",
+            'amount_type': 'percent',
+            'amount': 20.0
+        })
```

Comment:
```suggestion
```
I don't think we need to add a tax at all to replicate this issue, right? If not, then let's remove this and any other step irrelevant to the issue :)

Reply:
Don't forget to remove this too